cluster.go

```diff
@@ -281,6 +293,34 @@ func (c *Cluster) Sync() error {
 	return err
 }
 
+func isTopoContains(topo ClusterTopo, node ClusterNode) bool {
+	for _, t := range topo.Map() {
+		if reflect.DeepEqual(t, node) {
+			return true
+		}
+	}
+	return false
+}
+
+func (c *Cluster) traceTopoChanged(prevTopo ClusterTopo, newTopo ClusterTopo) {
+	if c.co.ct.TopoChanged != nil {
+		var Added []trace.ClusterNodeInfo
```

Comment:
Local variables should be lower case in Go. Same for all the other places.

Reply:
Yes, I'll change it.